resources/js/admin/settings/components/SettingsImport.vue

```diff
@@ -0,0 +1,127 @@
+<template>
+  <div>
+    <b-modal v-model="shown" :title="title" @hidden="onModalHidden">
+      <div>
+        <p>{{ $t('Please upload a settings file that was previously exported from ProcessMaker 4.') }}</p>
+        <p>{{ $t('All settings that are present on this system will be updated with your imported values.') }}</p>
+        <h5 class="text-center pt-3 pb-2" v-if="imported">
+          <i class="fas fa-check-circle text-success"></i> {{ $t("\{\{ count \}\} \{\{ plural \}\} updated.", {count: this.total, plural: this.plural}) }}
```

Comment:
Please move this translated label to a computed property or a method to improve the code reading.

Reply:
About, readability, it seems to me that it is more appropriate to place the strings where they pertain, in this case they are part of HTML. PM is placing translations in this way across all the application. For example in notifications.vue:

![image](https://user-images.githubusercontent.com/14875032/114713985-a7c17880-9cff-11eb-9059-d4ea3254d83e.png)

@caleeli  What do you think?